src/components/atoms/buttonAtom/ButtonAtom.jsx

```diff
@@ -0,0 +1,13 @@
+import css from './ButtonAtom.module.css';
+
+function ButtonAtom({ text, icon, handleClick }) {
+  return (
+    <button type="button" onClick={handleClick} className={css.button}>
+      {text} {icon}
+      <div className={css.button__horizontal}></div>
+      <div className={css.button__vertical}></div>
+    </button>
+  );
+}
```

Comment:
How you want to pass icon when u will use this component in code?

Reply:
@TN4HPEL3 I deleted this parameter, it used to be needed when I wanted to create an arrow in the button.